Phylum/README.md

```diff
@@ -0,0 +1,8 @@
+
+# Phylum
+
+Phylum's Sumologic app integrates Phylum's threat feed to enable Sumologic Users to alert on software supply chain attacks and malicious packages
```

Comment:
Could you please create README using the sample template https://github.com/SumoLogic/sumologic-public-partner-apps/blob/master/.github/README_TEMPLATE.md. You can also look into other app's README.md file for reference.

Reply:
@peterjmorgan - Can you fill this template out and replace here? Thanks!

Reply:
@himanshu219 @sumo-drew Updated!

Reply:
@himanshu219 are we set to get this merged so it can be available by the end of the month? Any more edits needed?